compiler_gym/service/proto/Proto.cc

```diff
@@ -0,0 +1,516 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include "compiler_gym/service/proto/Proto.h"
+
+#include <glog/logging.h>
+#include <stdint.h>
+
+#include <algorithm>
+#include <cstddef>
+#include <magic_enum.hpp>
+#include <optional>
+
+#define CG_PROTO_CHECK(cond) \
+  do {                       \
+    if (errorOnFalse) {      \
```

Comment:
I think `errorOnFalse` should be a macro argument

Reply:
Done.